views/components/Graphs/BarGraph.jsx

```diff
@@ -1,24 +1,24 @@
 import React from 'react'
-import { 
+import {
   VictoryBar,
   VictoryChart,
   VictoryAxis,
   VictoryTheme,
   VictoryStack,
   VictoryLegend,
   VictoryLabel,
-  VictoryTooltip
-} from 'victory';
+  VictoryTooltip,
+} from 'victory'
 
-import { graphStyles } from '../../styles/chart_styles';
+import { graphStyles } from '../../styles/chart_styles'
 
 const BarGraph = ({ graph }) => {
-  const { targetValuesMap } = graph
+  const groupDataByValue = (data) => Object.values(data)
```

Comment:
what do you think about calling `Object.values(graph.data).map(...)` directly below instead of splitting out this function? There isn't much logic in this function anymore which is making me think we could call `Object.values` directly.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/115/commits/181384a418ea7628c74eb97994f5992d4f1e3f71